charts/fybrik/values.yaml

```diff
@@ -314,7 +314,7 @@ katalogConnector:
 
   service:
     type: ClusterIP
-    port: 80
+    port: 8080
```

Comment:
I think the deployment will look for katalog-connector:80.
` CATALOG_CONNECTOR_URL: {{ .Values.coordinator.catalogConnectorURL | default (printf "%s-connector:80" .Values.coordinator.catalog) | quote }}`
You need to set coordinator.catalogConnectorURL to start with "http://" and point to the right port for notebook tests and the actual deployment.